x-pack/plugins/apm/server/routes/settings/custom_link.ts

```diff
@@ -17,6 +22,18 @@ import { getTransaction } from '../../lib/settings/custom_link/get_transaction';
 import { listCustomLinks } from '../../lib/settings/custom_link/list_custom_links';
 import { createRoute } from '../create_route';
 
+function isActiveGoldLicense(license: ILicense) {
+  return license.isActive && license.hasAtLeast('gold');
+}
+
+const INVALID_LICENSE = i18n.translate(
+  'xpack.apm.settings.customizeUI.customLink.forbidden',
+  {
+    defaultMessage:
+      "To create custom links, you must be subscribed to an Elastic Gold license or above. With it, you'll have the ability to create custom links to improve your workflow when analyzing your services.",
```

Comment:
Can we avoid the duplication?
https://github.com/elastic/kibana/blob/cbf2844d640346fa627a2ff193a773f2ed4e1457/x-pack/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/index.tsx#L102